admin/src/bench/db.rs

```diff
@@ -14,11 +14,13 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-pub type Key = [u8; 32];
-pub type Value = Vec<u8>;
-
+use parity_db::{Key, Value};
+// TODO useless??
```

Comment:
The bench could probably be change to remove this trait, I am keeping it in this PR to avoid changing code.

Reply:
It was introduced to be able to plugin in alternative databases and compare performance. Subdb was supported initially, but that was later dropped.